docs/source/man.rst

```diff
@@ -593,7 +593,7 @@ Uploading to Insights
 ~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I know it is from a previous PR but if you look at this file and see the insights header [here](https://github.com/quipucords/qpc/blob/6e9de098d47783af253096fd7f1b36f2a346a173/docs/source/man.rst#insights). It should be the same size as the Reports header

Reply:
Fixed: https://github.com/quipucords/qpc/blob/989567cbf9953376003d06348f0e35beeb9e9d40/docs/source/man.rst'